# makes the build fail on the build server somehow
# a2x -f manpage freedm.adoc
# a2x: ERROR: "xmllint" --nonet --noout --valid "/«PKGBUILDDIR»/dist/freedm.xml" returned non-zero exit status 127
# maybe because of invalid characters
Index: freedoom-0.9/dist/Makefile
===================================================================
--- freedoom-0.9.orig/dist/Makefile	2014-10-14 09:45:33.000000000 +0200
+++ freedoom-0.9/dist/Makefile	2014-10-14 17:52:41.190921185 +0200
@@ -1,6 +1,6 @@
 man-%: freedoom.adoc
-	sed -e s/freedoom/$*/ freedoom.adoc > $*.adoc
-	a2x -f manpage $*.adoc
+	#sed -e s/freedoom/$*/ freedoom.adoc > $*.adoc
+	#a2x -f manpage $*.adoc
 
 icon-freedm:
 	convert -trim +repage -extent 32x32 -gravity center \
Index: freedoom-0.9/Makefile
===================================================================
--- freedoom-0.9.orig/Makefile	2014-10-14 09:45:33.000000000 +0200
+++ freedoom-0.9/Makefile	2014-10-18 17:46:32.438276847 +0200
@@ -151,8 +151,8 @@
 install-%: $(WADS)/%.wad %.6 %.png
 	install -d "$(target)$(bindir)"
 	install -m 755 dist/freedoom "$(target)$(bindir)/$*"
-	install -d "$(target)$(mandir)/man6"
-	install -m 644 dist/$*.6 "$(target)$(mandir)/man6"
+	#install -d "$(target)$(mandir)/man6"
+	#install -m 644 dist/$*.6 "$(target)$(mandir)/man6"
 	install -d "$(target)$(waddir)"
 	install -m 644 $(WADS)/$*.wad "$(target)$(waddir)"
 	install -d "$(target)/share/applications"
